settings.gradle

```
@@ -25,4 +25,5 @@ include 'contentgrid-spring-boot-starter-annotations'
 include 'contentgrid-spring-boot-platform'
 include 'contentgrid-spring-data-rest'
 include 'contentgrid-spring-integration-events'
-include 'contentgrid-spring-querydsl'
\ No newline at end of file
+include 'contentgrid-spring-querydsl'
+include 'contentgrid-spring-swagger'
```

```suggestion
include 'contentgrid-spring-swagger-ui'
```

This only supports swagger UI, not the open api spec (a long time ago called swagger spec) itself.